Fighting Game Builder/build.py

```diff
@@ -38,13 +38,35 @@
     starting_items = []
 
     game_meta = y_data['meta']
-    playable_alias = game_meta.get("playable_alias", "Character")
+    game_name = game_meta["game_name"]
+    playable_alias = game_meta.get("playable_alias", "character")
     global_characters = y_data.get('characters', [])
-
+    # parse for global characters
+    for char in global_characters:
+        j_items.append({
+            "name": char,
+            "category": ["characters"],
+            "progression": True,
+            "count": 1
+        })
+    start_char_global = game_meta.get("starting_characters", 1)
+    if isinstance(start_char_global, int):
+        starting_items.append({
+            "item_categories": ["characters"],
+            "random": start_char_global
+        })
+    elif isinstance(game_meta["starting_characters"], list):
+        starting_items.append({
+            "items": game_meta["starting_characters"]
+        })
+    else:
+        raise ValueError("Mode values of starting_characters malformed.")
```

Comment:
This structure breaks if there are no global characters, since it will still try to add them to the starting character list